### ISSUE TITLE
Off-by-one error in testnet minimum difficulty block gap specification

### ISSUE THREAD

**Issue by teor2345** [ZcashFoundation/zebra#1276]
## Issue

The minimum difficulty block gap check is ambiguous in ZIP-205 and ZIP-208.

`zcashd` uses a strictly greater than check.

## Details

zcashd implements the minimum difficulty block gap using a strictly greater than test:
```c
            // If the new block's timestamp is more than 6 * block interval minutes
            // then allow mining of a min-difficulty block.
            if (pblock && pblock->GetBlockTime() > pindexLast->GetBlockTime() + params.PoWTargetSpacing(pindexLast->nHeight + 1) * 6)
                return nProofOfWorkLimit;
```
https://github.com/zcash/zcash/blob/514d86817990a1bf9578ee5fa2d01c34c6ca6035/src/pow.cpp#L33

But ZIP-205 specifies greater than or equal to:

> If the block time of a block from this height onward is at least 15 minutes after that of the preceding block, then the block is a minimum-difficulty block...
https://zips.z.cash/zip-0205#change-to-difficulty-adjustment-on-testnet

And ZIP-208 is ambiguous:

> ... the difficulty adjustment algorithm allows minimum-difficulty blocks, as described in 8, when the block time exceeds a given threshold. ...
>
> That is, if the block time of a block at height height ≥ 299188 is at least 6 · PoWTargetSpacing(height) seconds after that of the preceding block, then the block is a minimum-difficulty block...

https://zips.z.cash/zip-0208#minimum-difficulty-blocks-on-the-test-network

## Suggested Resolution

@dconnolly or @daira, can you update ZIP-205 and ZIP-208 to consistently say `exceeds` or `greater than`?

**Comment by dconnolly** [comment]
Done 🙏
https://github.com/zcash/zips/commit/806076c48c9834fd9941b940a32310d737975a3a

**Comment by teor2345** [comment]
We missed a `greater than` in this sentence in ZIP 208:

```
That is, if the block time of a block at height *height* ≥ 299188 is at least
```
https://github.com/zcash/zips/commit/806076c48c9834fd9941b940a32310d737975a3a#diff-3f875a100a287c2b47ce389870bdde636f2720429dc19a74d1b42b4e08e31962R187

**Comment by daira** [comment]
Thanks, I'll include this in zcash/zips#417.

**Comment by daira** [comment]
FIxed in zcash/zips#417.

**Comment by mpguerra** [comment]
@teor2345 Can we close this one now?
